packages/extensions/openapi-to-typespec/src/generate/generate-arm-resource.ts

```diff
@@ -1,5 +1,6 @@
 import { Dictionary } from "@azure-tools/openapi/v3";
 import { TypespecOperation, TspArmResource, TspArmResourceOperation } from "interfaces";
+import pluralize from "pluralize";
```

Comment:
Consider to enable linter and prettier to make code consistent.

Reply:
this package is in a mono repo and follow all the autorest lint and prettier config.